src/app/shared/services/switchboard-toastr.service.spec.ts

```diff
@@ -0,0 +1,120 @@
+import { TestBed } from '@angular/core/testing';
+
+import { SwitchboardToastr, SwitchboardToastrService } from './switchboard-toastr.service';
+import { ToastrModule, ToastrService } from 'ngx-toastr';
+import { BehaviorSubject } from 'rxjs';
+
+fdescribe('SwitchboardToastrService', () => {
+  let service: SwitchboardToastrService;
+  const toastrSpy = jasmine.createSpyObj('ToastrService',
+      [
+        'success',
+        'show',
+        'error',
+        'info',
+        'warning',
+      ]
+  );
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [{ provide: ToastrService, useValue: toastrSpy }],
+      imports: [ToastrModule.forRoot()]
```

Comment:
this import is not needed, because you are mocking already ToastrService. Importing entire module will take more time to build those tests.